### PR TITLE
Added re-apply changes sections in jobs

### PR DESCRIPTION
**Pull request by SeqLaz**
Added a section to re-apply changes in QFieldCloud in the jobs page

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3050 Steps to re-apply changes in QFieldCloud](https://app.clickup.com/t/2192114/QF-3050)

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 1]
```diff
@@ -81,6 +81,17 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
 - The project is too big and the job has failed to run.
 - There are hidden files and directories within the project that are preventing the normal work of QFieldCloud. Hidden files and directories are those starting with a leading dot (`.`).
 
+### Re-apply changes in QFieldCloud
+
+To reapply the changes, in QFieldCloud go to to project in the Changes section, the changes are order in reverse order by the time of the changes, find the changes you want to re-apply and in the right side in the Action column click in the dropdown list.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/jobs.en.md:86]:
```suggestion
To reapply the changes, in QFieldCloud go to the project in the Changes section, the changes are ordered in reverse order by the time of the changes, find the changes you want to re-apply and on the right side in the Action column click on the dropdown list.
```

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 2]
```diff
@@ -81,6 +81,17 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
...
+
+1. Re-apply Specific Changes: If you need to reapply specific changes, you can do so by selecting manually each change you want to re-apply and select "Re-apply".
+
+2. Re-apply Recent Changes: If you prefer to reapply  the most recent modifications made to the project, you can begin by identifying the initial change you want to restore, then select "Re-apply this and newer changes".
+
+Finally click the "Save all" button.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/jobs.en.md:92]:
```suggestion
Finally, click the "Save all" button.
```

File: documentation/reference/qfieldcloud/jobs.en.md (hunk 1 above)
Comment (suricactus) [reply documentation/reference/qfieldcloud/jobs.en.md:86]: Would be nice if we split this gigantic sentence into smaller digestable sentences.

**Review by SeqLaz — COMMENTED** [review]
Split into small sentence.

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 3]
```diff
@@ -81,6 +81,20 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
 - The project is too big and the job has failed to run.
 - There are hidden files and directories within the project that are preventing the normal work of QFieldCloud. Hidden files and directories are those starting with a leading dot (`.`).
 
+### Re-apply changes in QFieldCloud
+
+1. Access to the project.
```
Comment (why-not-try-calmer) [inline documentation/reference/qfieldcloud/jobs.en.md:86]:
```suggestion
1. Click on the project's name under  _My projects_.
```

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 4]
```diff
@@ -81,6 +81,20 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
...
+2. Go to the Changes section. (Changes are sorted in reverse order by time)
```
Comment (why-not-try-calmer) [inline documentation/reference/qfieldcloud/jobs.en.md:87]:
```suggestion
2. Go to the _Changes_ section. (Changes are sorted from latest to oldest)
```

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 5]
```diff
@@ -81,6 +81,20 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
...
+3. Find the specific changes you want to reapply.
+4. In the Action column on the right, click the dropdown list.
```
Comment (why-not-try-calmer) [inline documentation/reference/qfieldcloud/jobs.en.md:89]:
```suggestion
4. In the _Action_ column on the right, click on the dropdown list.
```

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 6]
```diff
@@ -81,6 +81,20 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
...
+5. Choose the desired action to reapply the changes.
+
+    - Re-apply Specific Changes: If you need to reapply specific changes, you can do so by selecting manually each change you want to re-apply and select "Re-apply".
```
Comment (why-not-try-calmer) [inline documentation/reference/qfieldcloud/jobs.en.md:92]:
```suggestion
    - Re-apply Specific Changes: If you need to reapply specific changes, select each change you want to re-apply and click __Re-apply__.
```

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 7]
```diff
@@ -81,6 +81,20 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
...
+    - Re-apply Recent Changes: If you prefer to reapply  the most recent modifications made to the project, you can begin by identifying the initial change you want to restore, then select "Re-apply this and newer changes".
```
Comment (why-not-try-calmer) [inline documentation/reference/qfieldcloud/jobs.en.md:93]:
```suggestion
    - Re-apply Recent Changes: If you prefer to reapply the most recent modifications made to the project, identify the initial change you want to restore, then select __Re-apply this and newer changes__.
```

File: documentation/reference/qfieldcloud/jobs.en.md [hunk 8]
```diff
@@ -81,6 +81,20 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
...
+
+Finally, click the "Save all" button.
```
Comment (why-not-try-calmer) [inline documentation/reference/qfieldcloud/jobs.en.md:95]:
```suggestion
Finally, click the __Save all__ button.
```

**Review by why-not-try-calmer — CHANGES_REQUESTED** [review]
This is good; a few suggestions though

**Review by why-not-try-calmer — APPROVED** [review]

**Pull request merged**